.github/actions/spelling/patterns.txt

```diff
@@ -17,3 +17,10 @@ data:[a-zA-Z=;,/0-9+-]+
 El proyecto .* diferentes
 # Package family names
 \b[-.A-Za-z0-9]+_[a-z0-9]{13}\b
+# URLs
+(?:(?:(?:ht|f)tp(?:s?))\://)?(?:www.|[a-zA-Z].)[a-zA-Z0-9\-\.]+\.(?:com|edu|gov|mil|net|org|biz|info|name|museum|us|ca|uk)(?:\:[0-9]+)*(?:/(?:$|[a-zA-Z0-9\.\,\;\?\'\\\+&%\$#\=~_\-]+))*
+# All tokens bounded by a slash, as these will be switches or parts of a file path for example
+(\\|/)\S+
+\S+(\\|/)
```

Comment:
Filenames are also regularly misspelled in my experience. You can of course ignore everything if you don't want the checking.

For microsoft/terminal, they have 3 files to specifically allow a small set of `\r`/`n`/`\t` names:
https://github.com/microsoft/terminal/tree/main/.github/actions/spelling/patterns

Reply:
Removed catch-all slash rules, and added a few specific patterns. Adding more pattern files like Terminal repo has would be a good idea I think, but I'll let the repo owners tackle that :)